openedx/core/djangoapps/content_staging/api.py

```diff
@@ -88,7 +88,7 @@ def get_staged_content_static_files(staged_content_id: int) -> list[StagedConten
     sc = _StagedContent.objects.get(pk=staged_content_id)
 
     def str_to_key(source_key_str: str):
-        if not str:
+        if not source_key_str:
```

Comment:
This was a clear bug found by the type check. 🐛  `str` here was the global string type :/